contracts/ligo/fa2_modules/admin/non_pausable_simple_admin.mligo

```diff
@@ -32,6 +32,14 @@ let fail_if_not_admin (storage : admin_storage) : unit =
   then failwith "NOT_AN_ADMIN"
   else unit
 
+(* Simplified version of 'fail_if_not_admin'.
+ * It's not worth using if at any place 'fail_if_not_admin' is required.
+*)
+let fail_if_not_admin_simple (storage : admin_storage) : unit =
```

Comment:
After the rebase this is now actually the same as `fail_if_not_admin`, so it can be removed